fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1132,7 +1132,7 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 				}
 				when(wait(pProxyCommitData->cx->onProxiesChanged())) {}
 				when(GetRawCommittedVersionReply v = wait(pProxyCommitData->master.getLiveCommittedVersion.getReply(
-				         GetRawCommittedVersionRequest(waitVersionSpan.context, debugID),
+				         GetRawCommittedVersionRequest(waitVersionSpan.context, debugID, invalidVersion),
```

Comment:
The sequencer will send the entire version vector  when replying to this request. Shall I add code to treat this as a special case and not send the version vector in the reply?

Reply:
I would add `// SOMEDAY: Special case this..etc`. And consider it when merging this project to trunk.

Reply:
Done.